templates/components/global-footer/global-footer.html.twig

```diff
@@ -6,7 +6,7 @@
 #}
 
 <footer id="footer__container">
-  <section class="contact-footer" style="background-image:url('{{ base_path ~ directory }}/img/optimized/footer-bg.jpg')";>
+  <section class="contact-footer b-lazy bg" data-src="{{ base_path ~ directory }}/img/optimized/footer-bg.jpg";>
```

Comment:
Hm, on my local on this branch I see this change in code, but don't see it reflected on my site: 
'<section class="contact-footer b-lazy bg b-loaded" ;="" data-bio-hit="1" style="background-image: url(&quot;/themes/stanford/matson/img/optimized/footer-bg.jpg&quot;);"></section>'

Reply:
Although I _do_ see the classes. Weird.